### ISSUE TITLE
Wrong error message when importing products with duplicate url_key

### ISSUE THREAD

**Issue by juanangel123** [magento/magento2#17170]
### Preconditions
1. Magento ver. 2.2.4.
2. PHP 7.0.30

### Steps to reproduce
1. Go to System > Import.
2. Download the sample product file.
3. Removed all rows except the first product.
4. Import the file.
6. Change the SKU of the product in the csv file.
5. Go again to Import page.
6. Error will appear.

### Expected result
It should show the sku and the url key of the wrong uploaded product.

### Actual result
![image](https://user-images.githubusercontent.com/4445993/43270331-18f34c8c-90f5-11e8-9a0e-2c1a003d11e9.png)

I've managed to debug and the error is in this class:

\Magento\CatalogImportExport\Model\Import\Product, line 2709

```php
$this->addRowError(ValidatorInterface::ERROR_DUPLICATE_URL_KEY, $rowNum);
```
Also, this class prints the same error in line 2416. I've managed to alter the code until I got the correct message:

 ```php
$urlKey = $entityData['request_path'];
$productUrlSuffix = $this->getProductUrlSuffix($storeId);
$urlPath = $urlKey . $productUrlSuffix;

$message = sprintf(
    $this->retrieveMessageTemplate(ValidatorInterface::ERROR_DUPLICATE_URL_KEY),
    $urlKey,
    $this->urlKeys[$storeId][$urlPath]
);
$this->addRowError(
    ValidatorInterface::ERROR_DUPLICATE_URL_KEY,
    $rowNum,
    $urlKey,
    $message
);
```

 But I don't fully understand all the variables involved to be sure that this code is actually ok.

**Comment by magento-engcom-team** [comment]
Hi @juanangel123. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@juanangel123 do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by juanangel123** [comment]
Sorry, just realized this issue is already fixed in branch 2.2-develop...

**Comment by engcom-backlog-nickolas** [comment]
Hello @juanangel123, thank you for your report. As you mentioned, this issue already fixed in 2.2-develop and 2.3-develop. So, I'm closing it. If you'd like to update it, please reopen the issue.
